src/compat/libc/stdlib/mbstowcs.c

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file
+ * @brief mbstowcs function.
+ *
+ * @see stdlib.h
+ *
+ * @date 01.11.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <defines/size_t.h>
+#include <defines/wchar_t.h>
+
+size_t mbstowcs(wchar_t *wcstring, const char *mbstring, size_t max) {
+	if (mbstring == NULL) {
```

Comment:
This check should be done after declarations

Reply:
actually, we don't need this check, cuz we check it in "while"

Reply:
Actually, this check is required. You should check `*mbstring` in loop, not `mbstring`.